koku/masu/database/report_manifest_db_accessor.py

```diff
@@ -51,6 +51,14 @@ def mark_manifest_as_completed(self, manifest):
             manifest.manifest_completed_datetime = self.date_accessor.today_with_timezone("UTC")
             manifest.save()
 
+    def update_number_of_files_for_manifest(self, manifest):
+        """Update the number of files for manifest."""
+        records = CostUsageReportStatus.objects.filter(manifest_id=manifest.id)
+        set_num_of_files = len(records)
```

Comment:
Can we use `.count()` here instead?

```
Note: If you only need to determine the number of records in the set (and don’t need 
the actual objects), it’s much more efficient to handle a count at the database level using 
SQL’s SELECT COUNT(*). Django provides a count() method for precisely this reason.
```
From [docs](https://docs.djangoproject.com/en/3.2/ref/models/querysets/).

Reply:
Done 👍